EpsilonDashboard/EpsilonDashboard.cpp

```diff
@@ -12,8 +14,22 @@ EpsilonDashboard::EpsilonDashboard(int& argc, char** argv)
     , communicationContainer_(new CommunicationContainer())
     , businessContainer_(new BusinessContainer(*communicationContainer_, *dataContainer_))
     , presenterContainer_(new PresenterContainer(*dataContainer_))
-    , viewContainer_(new ViewContainer(*presenterContainer_))
 {
+    QCommandLineParser parser;
+
+    QCommandLineOption raceModeOption("r");
+    parser.addOption(raceModeOption);
+
+    parser.process(*this);
+
+    int mode = DISPLAY;
```

Comment:
Let's treat the enum as the custom type instead of implicitly treating it as an int.
`Mode mode = DISPLAY`